plugins/runner/go.mod

```diff
@@ -6,65 +6,112 @@ require (
 	github.com/deepmap/oapi-codegen/v2 v2.2.0
 	github.com/docker/docker v26.1.4+incompatible
 	github.com/docker/go-connections v0.5.0
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/openclarity/vmclarity/core v0.7.1
 	github.com/openclarity/vmclarity/plugins/sdk-go v0.7.1
+	github.com/openclarity/vmclarity/scanner v0.7.1
```

Comment:
we shouldnt import scanner here since we import runner in the scanner itself (also, we should use replaces for internal code rather than from releases). this is also why the vet is failing

my suggestion is to move `scanner/utils/containerrootfs` to `utils/containerrootfs` module, and import `github.com/openclarity/vmclarity/utils` here instead, but using replace directives

Reply:
There is also a `utils/fsutils`, could that also be its new home?